test/e2e/toolchaincluster_test.go

```diff
@@ -174,3 +176,86 @@ func caBundle(bundle string) clusterOption {
 		c.Spec.CABundle = bundle
 	}
 }
+
+func TestForceMetricsSynchronization(t *testing.T) {
```

Comment:
May I know why is this test in toolchaincluster_test.go? it doesn't have anything in common with ToolchainCluster resources.

Reply:
because once again I got mixed with toolchainstatus 😬